sds_data_manager/lambda_code/SDSCode/opensearch_utils/document.py

```diff
@@ -130,7 +130,7 @@ def _update_contents(self):
         self.size = len(self.contents.encode("ascii"))
 
     def _validate_identifier(self, identifier):
-        if type(identifier) is str or type(identifier) is int:
+        if isinstance(identifier, (str, int)):
```

Comment:
I feel like it is subtle but it is impressive to me that an automatic check knows how to do this 😂

Reply:
The bot knew it was wrong, but it didn't auto-fix it, I manually updated these in the subsequent commit which is why I wanted you to review it too :)

Reply:
Ah ok, I was too quick to give the robot credit over you 😅 . Nice work.

Reply:
It does seem like something that wouldn't be too hard to update though... maybe you can add the fox portion to ruff ;)